README.md

```diff
@@ -280,14 +280,14 @@ files and start again.
 vcpkg is the easiest way to have dependencies installed. It downloads packages sources, headers and build libraries for whatever TRIPLET is set up (platform/arq).
 VCPKG maintains any installed package inside its own folder, allowing to have multiple vcpkg folder with different dependencies installed on each. This is also great because you don't have to install dependencies globally on your system.
 
-Follow next steps to install VCPKG and have it linked to cmake. The vcpkg repository is checked out at the ref in [vcpkg.yml](eng/pipelines/templates/steps/vcpkg.yml#L11). Azure SDK code in this version is known to work at that vcpkg ref.
+Follow next steps to install VCPKG and have it linked to cmake. The vcpkg repository is checked out at the ref in [vcpkg-ref.txt](eng/vcpkg-ref.txt). Azure SDK code in this version is known to work at that vcpkg ref.

```

Comment:
nit: Can we call this `tag`, `sha`, or `commit`, rather than ref?
```suggestion
Follow next steps to install VCPKG and have it linked to cmake. The vcpkg repository is checked out at the tag in [vcpkg-tag.txt](eng/vcpkg-tag.txt). Azure SDK code in this version is known to work at that vcpkg tag.

```

Reply:
"ref" most closely aligns with the name of the tag that we're using but it could also point to a branch name or a commit SHA... There is no word that means tag, commit SHA, or branch head, however each of those refers to a "commit" so I've updated to use "commit"